infra/gravity/cluster_status.go

```diff
@@ -173,13 +172,18 @@ func apiserverNode(ctx context.Context, nodes []Gravity) (api Gravity, other []G
 		return nil, nil, trace.Wrap(err, "failed to resolve %v", apiserver)
 	}
 
-	sort.Slice(nodes, func(i, j int) bool {
-		return nodes[i].Node().PrivateAddr() == apiserverAddr
-	})
-	api = nodes[0]
-	if api.Node().PrivateAddr() != apiserverAddr {
+	other = make([]Gravity, 0)
```

Comment:
This is redundant as `other` is technically already a slice with an underlying array of zero items.
You could instead create it already with the necessary capacity:
```go
other = make([]Gravity, 0, len(nodes)-1)
```

Reply:
Good call.  Fixed in 9d4be17.